examples/flutter_gallery/lib/demo/cupertino/cupertino_buttons_demo.dart

```diff
@@ -36,16 +36,16 @@ class _CupertinoButtonDemoState extends State<CupertinoButtonsDemo> {
                 padding: EdgeInsets.all(16.0),
                 child: Text(
                   'iOS themed buttons are flat. They can have borders or backgrounds but '
-                  'only when necessary.'
+                  'only when necessary.',
                 ),
               ),
               Expanded(
                 child: Column(
                   mainAxisAlignment: MainAxisAlignment.center,
-                  children: <Widget> [
-                    Text(_pressedCount > 0
-                        ? 'Button pressed $_pressedCount time${_pressedCount == 1 ? "" : "s"}'
-                        : ' '),
+                  children: <Widget>[
+                    Text(
+                      _pressedCount > 0 ? 'Button pressed $_pressedCount time${_pressedCount == 1 ? "" : "s"}' : ' ',
```

Comment:
i think this is an example where the two forks of the `?:` really need to be parallel rather than on the same line. Otherwise it's really hard to tell what the alternatives are, where they start and end.

Reply:
Agree. I wanted to add a rule to always split conditional statements in argument list with trailing comma. WDYT?